sql/log_event.cc

```diff
@@ -10199,6 +10200,26 @@ int Rows_log_event::do_apply_event(Relay_log_info const *rli) {
 
 end:
   if (get_flags(STMT_END_F)) {
+    if (table != nullptr && thd->slave_thread) {
+      /*
+        Set short description of RBR event to be used by slow query log.
+       */
+      std::ostringstream log_query;
+      log_query << get_type_str() << " " << table->s->db.str << "."
+                << table->s->table_name.str;
```

Comment:
Not a strong opinion: Shouldn't we quote db and table names?

Something like ``Write_rows `db_name`.`table_name` ``

Reply:
Don't have strong opinion as for this as well. Added quotes.